ckanext/canada/tables/suppliervax.yaml

```diff
@@ -0,0 +1,424 @@
+dataset_type: suppliervax
+target_dataset: suppliervax
+
+title: Reporting on COVID-19 Vaccination Requirement for Supplier Personnel 
+notes: Access, upload and modify the COVID-19 Vaccination Requirement for Supplier Personnel reporting for your organization
+
+
+upload_warn:
+  en: "placeholder if required"
+  fr: "placeholder if required"
+
+
+
+template_version: 3
+
+portal_type: info
+collection: pd
+
+resources:
+- title: Contract Data - Reporting on COVID-19 Vaccination Requirement for Supplier Personnel
+  resource_name: contract-data
+  create_form: true
+  edit_form: true
+  fields:
+
+  # 3.1
+  - datastore_id: c_report_date
```

Comment:
oh, `c_` is for "contract", I thought you were naming all your columns starting with `c_`